README.md

```diff
@@ -283,6 +286,72 @@ const NewComponent = exclude(Component);
 
 Stops `prepare` traversal at `Component`. Useful for optimizing the `prepare` traversal to visit the minimum number of nodes.
 
+#### useService
+
+*React Hooks were introduced in React v16.8. Make sure you are using a compatible version.*
+
+```js
+import {useService} from 'fusion-react';
+import {ExampleToken} from 'fusion-tokens';
+
+function Component() {
+  const service = useService(ExampleToken);
+  return (
+    <button onClick={service}>Invoke Service</button>
+  );
+}
+```
+
+- `token: Token` - Required. The token used to lookup the registered plugin.
```

Comment:
Nitpick: we could avoid `any` here by using the generic on `Token`.  Something like:

```
`token: Token<TService>` - Required.  The token used to look up the registered plugin that resolves to `TService`.
`service: TService` - ...
```

Reply:
Makes sense